application/views/admin/global_settings/_emailTestResults.php

```diff
@@ -0,0 +1,18 @@
+<div class="pagetitle h3"><?php eT("Test Email Settings");?></div>
+        
+<div class="row" style="margin-bottom: 100px">
+    <div class="col-lg-12">
+        <div class="jumbotron message-box">
+            <h2><?php eT("Test Email Results"); ?></h2>
+            <p class="lead"><?php eT("If errors are showing up you might have to change configuration settings and retest."); ?></p>
+            <p>
```

Comment:
maildebug can have tag, i don't remind if we add pre tag automtically (in fact : a lot of fix must be done to mail system).

But here : need a div.

Maybe : 
```
<?php if($success) { ?>
    <p class="h4 text-success"><?php echo $message;?></p>
<?php } else { ?>
    <p class="h4text-danger"><?php echo $message;?></p>
    <div> <?php echo $maildebug ?></div>
<?php } ?>
```

But remind : i'm not the one who can merge :)